fineract-provider/build.gradle

```diff
@@ -21,11 +21,11 @@ Run as:
 gradle clean tomcatrunwar
 '''
 
-project.ext.springBootVersion = '2.1.9.RELEASE'
-project.ext.springVersion = '5.1.10.RELEASE'
-project.ext.springOauthVersion = '2.3.6.RELEASE'
-project.ext.jerseyVersion = '1.17'
-project.ext.springDataJpaVersion = '2.2.0.RELEASE' // also change spring-boot-gradle-plugin version below
+// project.ext.springBootVersion = '2.1.9.RELEASE'
+// project.ext.springVersion = '5.1.10.RELEASE'
+// project.ext.springOauthVersion = '2.3.6.RELEASE'
```

Comment:
Let's just completely remove these 3 lines of Spring versions? (They are only used once further down.)